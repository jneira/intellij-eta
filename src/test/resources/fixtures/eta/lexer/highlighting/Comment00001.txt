WHITE_SPACE ('\n\n')
ITlineComment ('--\n')
ITlineComment ('-- Pretend long copyright notice is here.\n')
ITlineComment ('--\n')
WHITE_SPACE ('\n')
ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Comment1')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITvarid ('main')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
ITvarid ('print')
WHITE_SPACE (' ')
ITstring ('"')
ITstring ('Hello, complicated world!')
ITstring ('"')
WHITE_SPACE ('\n')
