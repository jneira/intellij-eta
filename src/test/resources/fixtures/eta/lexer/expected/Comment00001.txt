WHITE_SPACE ('\n\n')
ITlineComment ('--')
WHITE_SPACE ('\n')
ITlineComment ('-- Pretend long copyright notice is here.')
WHITE_SPACE ('\n')
ITlineComment ('--')
WHITE_SPACE ('\n\n')
ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Comment1')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITvocurly ('')
WHITE_SPACE ('')
ITvarid ('main')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
ITvarid ('print')
WHITE_SPACE (' ')
ITstring ('"Hello, complicated world!"')
WHITE_SPACE ('\n')
ITsemi ('')